qiita_db/environment_manager.py

```diff
@@ -101,18 +103,21 @@ def _add_ontology_data(conn):
             raise IOError("Error: Could not fetch ontologies file from %s" %
```

Comment:
This part should probably also be in the TRN, since it can fail and cause rollback.